pkg/apis/certificates/validation/validation.go

```diff
@@ -58,6 +60,52 @@ func ValidateCertificateSigningRequest(csr *certificates.CertificateSigningReque
 	if len(csr.Spec.Usages) == 0 {
 		allErrs = append(allErrs, field.Required(specPath.Child("usages"), "usages must be provided"))
 	}
+	if csr.Spec.SignerName == nil {
+		allErrs = append(allErrs, field.Required(specPath.Child("signerName"), "signerName must be provided"))
+	} else {
+		// ensure signerName is of the form domain.com/something and up to 571 characters
```

Comment:
@liggitt I remember you had a particular format in mind that you wanted to support. This is a good spot to document it.

Reply:
`<dnsSubdomain>/<dnsLabel>.<dnsSubdomain>` I think